airflow/__init__.py

```diff
@@ -44,23 +44,8 @@
 
 settings.initialize()
 
-login = None  # type: Optional[Callable]
+from airflow.plugins_manager import integrate_plugins
```

Comment:
Any reason not to have this import at the top with the normal block?

Because of L31 we've diabled "wrong-import-position" for the entire fil

Reply:
Oh, cos importing plugins mananger needs settings to have been loaded. Right.

_somewhat_ off scope, so this can probably stay as it is for now, but we could move all of the top level code in airflow.plugins_manager in to this new `integrate_plugins` method.

Reply:
Yep. I think I have done it in the follow-up change where i add a lot of typing and restructured plugins_manager.